assets/pages/bundles-landing/components/Bundle.jsx

```diff
@@ -0,0 +1,48 @@
+// ----- Imports ----- //
```

Comment:
~~I'm surprised that the components sub-directory is nested under the pages. Is it expected that components won't be shared across different pages?~~